### PR TITLE
ASU-765 Finalize initial conf for production

### PR DESCRIPTION
**Pull request by tomijarvi**
There were some of the production configurations already done to this
repository so this finalizing the initial configuration for production
environments.

The code bases to Open City Profile's implementations:
https://github.com/City-of-Helsinki/open-city-profile

### CODE REVIEW

File: .prod/on_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+python /app/manage.py migrate --noinput
+
+if [[ "$SEED_DEVELOPMENT_DATA" = "1" ]]; then
+    python /app/manage.py seed_data --development --superuser
```
Comment (jmp) [inline .prod/on_deploy.sh:6]: I think the `seed_data` management command is missing. Should we also add it to this PR?

File: .prod/uwsgi.ini [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+static-map = /static=/app/static
+uid = nobody
+gid = nogroup
+buffer-size = 32768
+master = 1
+processes = 2
+threads = 2
+; don't log readiness and healthz endpoints
+route = ^/readiness$ donotlog:
+route = ^/healthz$ donotlog:
```
Comment (jmp) [inline .prod/uwsgi.ini:14]: We don't have these endpoints (yet?), so it might be okay to leave these out. 🙂

**Review by jmp — COMMENTED** [review]
I left some comments. 🙂

File: .prod/on_deploy.sh (hunk 1 above)
Comment (tomijarvi) [reply .prod/on_deploy.sh]: I removed it completely as we don't have the command and we don't need it, at least yet. I probably mixed this "built-in" command with other platform's command. 😄

File: .prod/uwsgi.ini (hunk 2 above)
Comment (tomijarvi) [reply .prod/uwsgi.ini]: This is removed as well, we don't have these at least yet.

**Comment by tomijarvi** [comment]
@jmp Ready for another review.

**Review by jmp — APPROVED** [review]
Looks good! 😊

**Review by genie9 — APPROVED** [review]

**Pull request merged**